fetch.sh

```diff
@@ -200,7 +220,9 @@ cpuspeed () {
 
 # Memory (Configurable with "-M" and "--memory" at launch)
 # Print the total amount of ram and amount of ram in use
-memory=$(free -m | awk '/Mem:/ {printf $3 "MB / " $2 "MB"}')
+usedmem=$(free -m | awk '/cache:/ {print $3}')
```

Comment:
My bad.

Also I just compared the output from free on my pc and it differs to yours which means we'll have to some sort of detection or get the memory a different way.

This line doesn't output anything on my pc because of above.

    usedmem=$(free -m | awk '/cache:/ {print $3}')